NOTES_ON_CODEPUSH.md

```diff
@@ -29,33 +29,33 @@ required less developer effort to maintain.
 ## Code push approaches
 
 Many approaches one could take:
-1. Compile Release Flutter with DartVM in JIT mode
-   Implementation Difficulty: Medium
-   Application Performance: Unreliable (due to JIT compiles), but good once warmed up.
+1. Compile Release Flutter with DartVM in JIT mode\
```

Comment:
TIL \ is a way of forcing a newline: https://stackoverflow.com/questions/36583502/how-to-force-a-linebreak